core/pom.xml

```diff
@@ -100,7 +100,12 @@
             <version>${project.version}</version>
             <scope>test</scope>
         </dependency>
-
+        <dependency>
+            <groupId>org.apache.directory.server</groupId>
+            <artifactId>apacheds-all</artifactId>
```

Comment:
What is included in `all`? I ask because we list each dependency explicitly in Kafka.